server/mongoDB Commands/exportDB.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose');
+const fs = require('fs');
+
+const Bob = require('../models/bob');
+const Flavor = require('../models/flavor');
+const Tag = require('../models/tag');
+
+mongoose.Promise = global.Promise;
```

Comment:
The usage of this promise is all under the hood, yes? My understanding is that you're making the underlying promises consistent with the promises we write. This sounds good with me but it'd be good to add a comment explaining this.